0800-appendix-58067-gramophone.Rmd

```diff
@@ -209,14 +216,19 @@ Electrofishing was conducted in 2022 with results summarised in Tables
 \@ref(tab:tab-fish-site-58067) - \@ref(tab:tab-fish-dens-58067) and
 Figure \@ref(fig:plot-fish-box-58067).
 
-In 2023 fish sampling was conducted below the observed falls using dip nets. Six Coho were caught, 2 fry (FL ranging from 65-67mm), and 4 par (FL ranging from 70-90mm), see photos \@ref(fig:fish). Since Coho were captured below the falls but none at crossing `r as.character(my_site)` (during the 2022 sampling), it is presumed that the falls are a barrier to Coho passage. 
+In 2023 fish sampling was conducted below the observed falls using dip nets. Six Coho parr were captured and had a wide range of sizes (FL ranging from 65-90 mm) which suggests that spawning may have occurred at different times or rearing environments were more favourable (more food, warmer) for some, see figures \@ref(fig:fish1), \@ref(fig:fish2), \@ref(fig:fish3). Since Coho were captured below the falls but none at crossing `r as.character(my_site)` during the 2022 sampling, it is presumed that the falls are a barrier to Coho passage. 
```

Comment:
ok good. dipnetting mentioned here.  

I will give this memo a massage and see if it feels right to mention earlier as well....

Not a falls

>during the 2022 sampling, it is presumed 

tense is mixed here (past and current)... pain in the butt but worth trying to get right. A constant struggle....